### DISCUSSION TITLE
Storing files in Azurite (File Shares)

### DISCUSSION THREAD

**Discussion by 51imyyy** [Azure/Azurite#2355] [Q&A]
Hey, i am storing files in Azure File Shares. For local development i found this project, but i did not see file shares, only Blob Containers, Queues and Tables. Am i missing something or is there currently no file shares available? If not would it be possible to add this?

**Comment by blueww** [comment]
@51imyyy 

Thanks for contact us!

File service is still not supported in Azurite. 
You can find only blob/Queue/Table are supported in the Readme file: https://github.com/Azure/Azurite#:~:text=Blob%2C%20Queue%2C%20and%20Table%20(preview)

Besides that, we already has a issue to track this request: https://github.com/Azure/Azurite/issues/113

**Reply by 51imyyy** [reply]
Thanks for the fast response! 
Good to know that there is already an issue
